dietpi/dietpi-software

```diff
@@ -13262,6 +13339,16 @@ _EOF_
 
 		fi
 
+		software_id=186 # Box86
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
+
+			Banner_Uninstalling
+			rm /usr/local/bin/box86
+			rm /etc/binfmt.d/box86.conf
+			rm -R /usr/lib/i386-linux-gnu/
```

Comment:
Could that be a bit dangerous, if there are any other i386 libraries installed? Although how and what for would that be when not for Box86. Not sure, or is there some "uninstall" make command where we can pull the files from that were actually installed together with Box86?

Reply:
There is, and it's basically what I used. `cmake -P cmake_uninstall.cmake -- -n` lists the files, including specific libraries in `/usr/lib/i386-linux-gnu/`.  I'll remove those instead of the whole directory.